src/features/styles/theme/inputs/text.styles.scss

```diff
@@ -31,13 +31,14 @@
                 outline: none;
                 ~ .text_input_label {
                     inset-block-start: -8px;
+                    color: $color-black;
```

Comment:
This css will change the all inputs. Are we changing all?

Reply:
we are using this in only one place in home page its a single field, I thought every fields need to behave like same , only the label color is going to change so I changed this